src/gbd_mapping_generator/data.py

```diff
@@ -334,25 +337,19 @@ def get_risk_data():
             paf_of_one_causes = []
 
         if paf_calculation_type in ['continuous', 'categorical', 'custom']:
-            missing_rr = risk['missing_rr']
-            rr_less_than_1 = not risk['rr_less_than_1']
+            rr_exists = risk['rr_exists']
+            rr_in_range = risk['rr_in_range']
         else:
-            missing_rr = None
-            rr_less_than_1 = None
+            rr_exists = None
+            rr_in_range = None
 
         violated_restrictions = []
-        if risk['exposure_age_restriction_violated']:
-            violated_restrictions.append("age_restriction_violated_by_exposure")
-        if risk['exposure_sex_restriction_violated']:
-            violated_restrictions.append("sex_restriction_violated_by_exposure")
-        if risk['rr_age_restriction_violated']:
-            violated_restrictions.append("age_restriction_violated_by_rr")
-        if risk['rr_sex_restriction_violated']:
-            violated_restrictions.append("sex_restriction_violated_by_rr")
-        if risk['paf_age_restriction_violated']:
-            violated_restrictions.append("age_restriction_violated_by_paf")
-        if risk['paf_sex_restriction_violated']:
-            violated_restrictions.append("sex_restriction_violated_by_paf")
+        for restr_type in ["exposure_age_restriction_violated", "exposure_sex_restriction_violated",
```

Comment:
it might be easier to just pull all columns from risk that have the word restriction in them?